src/explorer/siteNodeBase.ts

```diff
@@ -247,4 +234,21 @@ export class SiteNodeBase extends NodeBase {
         const serverFarmId = util.parseAzureResourceId(this.site.serverFarmId.toLowerCase());
         return await this.webSiteClient.appServicePlans.get(serverFarmId.resourcegroups, serverFarmId.serverfarms);
     }
+
+    private async updateScmType(node: SiteNodeBase, config: WebSiteModels.SiteConfigResource): Promise<boolean> {
+        let input;
+        const oldScmType = config.scmType;
+        const updateConfig = config;
+        updateConfig.scmType = 'LocalGit';
+        input = await window.showWarningMessage(`Deployment source for "${node._siteName}" is set as "${oldScmType}".  Change to "LocalGit"?`, 'Yes');
+        if (input) {
```

Comment:
Personally I prefer this to be explicit: if (input === 'Yes') - in case in the future someone adds a new option but forgets to update the if line.